src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEFieldSymbol.cs

```diff
@@ -64,30 +66,32 @@ public bool TryGetFlowAnalysisAnnotations(out FlowAnalysisAnnotations value)
         }
 
         private readonly FieldDefinitionHandle _handle;
+#nullable disable // The compiler cannot figure out that _name is assigned in the constructor. https://github.com/dotnet/roslyn/issues/39166
```

Comment:
Filed [https://github.com/dotnet/roslyn/issues/41296](https://github.com/dotnet/roslyn/issues/41296) for this one

Reply:
I believe the issue is at least partially this: #41273